shared/js/content-scope/utils.js

```diff
@@ -130,19 +133,43 @@ export function defineProperty (object, propertyName, descriptor) {
     }
 }
 
+function camelcase (dashCaseText) {
+    return dashCaseText.replace(/-(.)/g, (match, letter) => {
+        return letter.toUpperCase()
+    })
+}
+
 export class DDGProxy {
-    constructor (objectScope, property, proxyObject) {
+    constructor (featureName, objectScope, property, proxyObject) {
         this.objectScope = objectScope
         this.property = property
+        this.featureName = featureName
```

Comment:
`featureName`'s only use seems to be for `postDebugMessage` where it has to be camelCased. Why not just set it in camelCase originally?

Reply:
`shouldExemptMethod` works on the raw name instead which works just on the passed raw names.

Reply:
We could instead camelcase on injection into the content script but we'd just move the problem I think as the inclusion of the files requires the kabab case.